CHANGELOG.md

```diff
@@ -14,6 +14,26 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [3.0.1] - 2019-04-26
+### Added
+- Code formatting tools for pom.xml and java source files
+- Support for use with Configuration as Code plugin, with test dependencies on configuration-as-code
+  version 1.9
+- Secret used during integration test can be provided through GOOGLE_CREDENTIALS_FILE environment
+  variable which should be a path to the secret file
+- Documentation transferred from wiki.jenkins.io and stored in source
+
+### Changed
+- Use unique IDs for each instance
```

Comment:
Can we change this to unique ID's for each cloud. The commit does say unique ID for each instance, but I think it was more meant to be unique instance ID for each cloud.

Reply:
Good catch, I'll update it.

Reply:
Done.